sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2050,4 +2050,60 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       }
     }
   }
+
+  test("SPARK-18355 Use Spark schema to read ORC table instead of ORC file schema") {
```

Comment:
Improve the test case for checking the other formats?

Reply:
since it depends on the CONVERT_METASTORE_XXX conf, maybe also test parquet.

Reply:
Yep. I'll add `parquet`, too.